### PR TITLE
[CLI-2857] Remove flag resolver

### PR DESCRIPTION
**Pull request by sgagniere**
Release Notes
-------------

Breaking Changes
- The API key and secret are now required arguments for `confluent api-key store`
- The `--config` flag no longer accepts pipes or files for `confluent secret file add`, `confluent secret file remove`, and `confluent secret file update`
- The `--passphrase` and `--passphrase-new` flags of `confluent secret file rotate` are now required and no longer accept pipes or files
- The `--passphrase` flag of `confluent secret master-key generate` is now required and no longer accepts pipes or files

New Features
- PLACEHOLDER

Bug Fixes
- PLACEHOLDER

Checklist
---------
- [x] Leave this box unchecked if features are not yet available in production

What
----

References
----------

Test & Review
-------------

### CODE REVIEW

File: internal/api-key/command.go [hunk 1]
```diff
@@ -38,17 +37,14 @@ const (
...
 		Use:         "api-key",
 		Short:       "Manage API keys.",
 		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
 	}
 
-	c := &command{
-		AuthenticatedCLICommand: pcmd.NewAuthenticatedCLICommand(cmd, prerunner),
-		flagResolver:            resolver,
-	}
+	c := &command{AuthenticatedCLICommand: pcmd.NewAuthenticatedCLICommand(cmd, prerunner)}
```
Comment (brianstrauch) [inline internal/api-key/command.go:47]:
```suggestion
	c := &command{pcmd.NewAuthenticatedCLICommand(cmd, prerunner)}
```

File: internal/context/command.go [hunk 2]
```diff
@@ -10,21 +10,17 @@ import (
...
 		Aliases: []string{"ctx"},
 		Short:   "Manage CLI configuration contexts.",
 		Long:    "Manage CLI configuration contexts. Contexts define the state of a Confluent Cloud or Confluent Platform login.",
 	}
 
-	c := &command{
-		CLICommand: pcmd.NewAnonymousCLICommand(cmd, prerunner),
-		resolver:   resolver,
-	}
+	c := &command{CLICommand: pcmd.NewAnonymousCLICommand(cmd, prerunner)}
```
Comment (brianstrauch) [inline internal/context/command.go:23]:
```suggestion
	c := &command{pcmd.NewAnonymousCLICommand(cmd, prerunner)}
```

File: internal/secret/command_file_add.go [hunk 3]
```diff
@@ -27,12 +27,7 @@ func (c *command) newAddCommand() *cobra.Command {
 }
 
 func (c *command) add(cmd *cobra.Command, _ []string) error {
-	config, err := cmd.Flags().GetString("config")
-	if err != nil {
-		return err
-	}
-
-	newConfigs, err := c.getConfigs(config, "config properties", "", false)
+	configs, err := cmd.Flags().GetString("config")
```
Comment (brianstrauch) [inline internal/secret/command_file_add.go:30]:
```suggestion
	config, err := cmd.Flags().GetString("config")
```

File: internal/secret/command_file_remove.go [hunk 4]
```diff
@@ -26,12 +26,7 @@ func (c *command) newRemoveCommand() *cobra.Command {
 }
 
 func (c *command) remove(cmd *cobra.Command, _ []string) error {
-	config, err := cmd.Flags().GetString("config")
-	if err != nil {
-		return err
-	}
-
-	removeConfigs, err := c.getConfigs(config, "config properties", "", false)
+	configs, err := cmd.Flags().GetString("config")
```
Comment (brianstrauch) [inline internal/secret/command_file_remove.go:29]:
```suggestion
	config, err := cmd.Flags().GetString("config")
```

File: internal/secret/command_file_rotate.go [hunk 5]
```diff
@@ -22,13 +22,15 @@ func (c *command) newRotateCommand() *cobra.Command {
 	}
 
 	cmd.Flags().String("local-secrets-file", "", "Path to the encrypted configuration properties file.")
+	cmd.Flags().String("passphrase", "", `Master key passphrase.`)
+	cmd.Flags().String("passphrase-new", "", `New master key passphrase.`)
```
Comment (brianstrauch) [inline internal/secret/command_file_rotate.go:26]:
```suggestion
	cmd.Flags().String("passphrase", "", "Master key passphrase.")
	cmd.Flags().String("passphrase-new", "", "New master key passphrase.")
```

File: internal/secret/command_master_key_generate.go [hunk 6]
```diff
@@ -20,12 +18,8 @@ func (c *command) newGenerateFunction() *cobra.Command {
 		RunE:  c.generate,
 		Example: examples.BuildExampleString(
 			examples.Example{
-				Text: `Pipe the passphrase from stdin:`,
-				Code: "confluent secret master-key generate --local-secrets-file /path/to/secrets.txt --passphrase -",
-			},
-			examples.Example{
-				Text: `Read the passphrase from the file "/User/bob/secret.properties":`,
-				Code: "confluent secret master-key generate --local-secrets-file /path/to/secrets.txt --passphrase @/User/bob/secret.properties",
+				Text: `Generate a master key`,
```
Comment (brianstrauch) [inline internal/secret/command_master_key_generate.go:21]:
```suggestion
				Text: "Generate a master key.",
```

File: internal/api-key/command.go (hunk 1 above)
Comment (sgagniere) [reply internal/api-key/command.go:47]: Can't do this one because this command struct also has a keystore.

**Review by brianstrauch — APPROVED** [review]

**Pull request merged**